app/views/users/_organization_user.html.erb

```diff
@@ -17,10 +17,11 @@
         <ul class="dropdown-menu">
           <li>
             <%=
-            edit_button_to(
```

Comment:
The issue here is that this partial is used for both superadmin views and org admin view. For admin view, we need to keep the edit button. We have to change it so org admins don't see that edit button and instead see the promote button. So this view has to either be duplicated and separated out, or we need to add additional if statements inside it.

Reply:
That's been addressed.